flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/table/MysqlConnectorCharsetITCase.java

```diff
@@ -387,7 +387,7 @@ public void testCharset() throws Exception {
         }
         assertEqualsInAnyOrder(
                 Arrays.asList(binlogExpected), fetchRows(iterator, binlogExpected.length));
-        result.getJobClient().ifPresent(client -> client.cancel());
+        result.getJobClient().get().cancel().get();
```

Comment:
Occasionally Db2 ITCase will fail because of trying to cancel a job which isn't running (#2345). Will same thing happen here?

Reply:
Thanks for your advice, @yuxiqian . As an unbounded streaming job, this job  should not stop before  can it.

To be honest, I am a little confused that in db2 ItCase, why you skip job when a job isn't running. Does it a bounded job? If not, isn't it a wrong bahavior?